### PR TITLE
Release 9.0.0

### PR DESCRIPTION
**Pull request by r0mant**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,128 @@
 # Changelog
 
+## 9.0.0
+
+Teleport 9.0  is a major release that brings:
+
+- Teleport Desktop Access GA
+- Teleport Machine ID Preview
+- Various additions to Teleport Database Access
+- Moderated Sessions for Server Access
```
Comment (zmb3) [inline CHANGELOG.md:10]: Server and kube, right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,128 @@
...
+
+Desktop Access GA adds  support for clipboard, session recording, and
```
Comment (zmb3) [inline CHANGELOG.md]:
```suggestion
Desktop Access adds support for clipboard sharing, session recording, and
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,128 @@
...
+per-session MFA.
+
+Teleport Machine ID Preview extends identity-based access to machines. It's the
+easiest way to issue, renew, and manage SSH and X.509 certificates for service
+accounts, microservices, CI/CD automation and all other forms of
+machine-to-machine access.
+
+Database Access brings self-hosted Redis support, RDS MariaDB (10.6 and higher)
+support,  auto-discovery for Redshift clusters, and auto-IAM configuration
```
Comment (zmb3) [inline CHANGELOG.md]:
```suggestion
support, auto-discovery for Redshift clusters, and auto-IAM configuration
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,131 @@
...
+## 9.0.0
+
+Teleport 9.0 is a major release that brings:
+
+- Teleport Desktop Access GA
+- Teleport Machine ID Preview
+- Various additions to Teleport Database Access
+- Moderated Sessions for Server Access
+
+Desktop Access GA adds support for clipboard, session recording, and
```
Comment (zmb3) [inline CHANGELOG.md:12]:
```suggestion
Desktop Access adds support for clipboard, session recording, and
```

I don't think we need to say `GA` so much.

**Review by zmb3 — APPROVED** [review]

**Review by russjones — APPROVED** [review]

**Pull request merged**